gm4_lightning_in_a_bottle/data/gm4_lightning_in_a_bottle/functions/lightning/check.mcfunction

```diff
@@ -0,0 +1,32 @@
+# Creates and empowers certain mobs and creates Bottles of Lightning
+# @s = lightning bolt
+# at @s
+# run from lightning/locate
+
+# create Bottles of Lightning from Thick Potions in Brewing Stands under (or above) Lightning Rods
+execute if block ~ ~-1 ~ lightning_rod[facing=up] if block ~ ~-2 ~ brewing_stand positioned ~ ~-2 ~ run function gm4_lightning_in_a_bottle:brewing_stand/check_fuel
+execute if block ~ ~-1 ~ lightning_rod[facing=down] if block ~ ~ ~ brewing_stand run function gm4_lightning_in_a_bottle:brewing_stand/check_fuel
+
+# Lightning-based mob transformation
+# Evokers -> Illusioners
+execute align xyz positioned ~-3 ~-2 ~-3 as @e[type=evoker,dx=6,dy=11,dz=6] at @s run function gm4_lightning_in_a_bottle:entity/illusioner
+# Pillagers -> Evokers
+execute align xyz positioned ~-3 ~-2 ~-3 as @e[type=pillager,dx=6,dy=11,dz=6] at @s run function gm4_lightning_in_a_bottle:entity/evoker
```

Comment:
Personally, I'm still not a huge fan of this transformation. This isn't a blocking issue but we should see what other team members think.